app/talk/discussion-new-form.cjsx

```diff
@@ -68,13 +69,11 @@ module?.exports = React.createClass
       <div className="talk-board-new-discussion">
         <h2>Create a disussion +</h2>
         {if not @props.boardId
-          <PromiseRenderer promise={@props.focusImage.get('project')}>{(project) =>
-            <PromiseRenderer promise={talkClient.type('boards').get(section: "project-#{project.id}")}>{(boards) =>
-              <div>
-                <h2>Board</h2>
-                {boards.map @boardRadio}
-              </div>
-            }</PromiseRenderer>
+          <PromiseRenderer promise={talkClient.type('boards').get(section: projectSection(@props.focusImage))}>{(boards) =>
```

Comment:
Is `@props.focusImage` a project resource? I can't find anyplace where it's actually passed into a DiscussionNewForm.